crates/runtime/src/traphandlers.rs

```diff
@@ -171,8 +221,12 @@ pub struct CallThreadState {
     jmp_buf: Cell<*const u8>,
     handling_trap: Cell<bool>,
     signal_handler: Option<*const SignalHandler<'static>>,
-    prev: Cell<tls::Ptr>,
+    pub(crate) prev: Cell<tls::Ptr>,
```

Comment:
Is this still necessary with the `.iter()` method below?

Reply:
It is used in a debug assertion inside `Backtrace::trace_with_trap_state`. Maybe I will make the `backtrace` module a submodule of `traphandlers`, since it is the only thing that wants access to these otherwise-private fields.